### PR TITLE
PerfView: Add version 3.0.1

### PR DESCRIPTION
**Pull request by rzikm**
This PR adds manifest for [PerfView](https://github.com/Microsoft/perfview).

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/perfview.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "description": "PerfView is a CPU and memory performance-analysis tool",
+    "homepage": "https://github.com/Microsoft/perfview",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/microsoft/perfview/releases/download/v3.0.1/PerfView.exe",
+            "hash": "2e00ad19de6abc5f776f5a3539748962778b37db0472913a54b8eb0ca06a2286"
+        }
+    },
+    "bin": "PerfView.exe",
```
Comment (rashil2000) [inline bucket/perfview.json:12]: Does the binary support commandline arguments?

File: bucket/perfview.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "shortcuts": [
+        [
+            "PerfView.exe",
+            "PerfView"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Microsoft/perfview"
+    },
```
Comment (rashil2000) [inline bucket/perfview.json:21]:
```suggestion
    "checkver": "github",
```

**Comment by rashil2000** [comment]
Where does PerfView store its configuration files?

File: bucket/perfview.json (hunk 1 above)
Comment (rzikm) [reply bucket/perfview.json:12]: yes, it can even be started headless/without GUI

**Comment by rzikm** [comment]
> Where does PerfView store its configuration files?

I don't think it stores any config files, there is no mention of them in the docs and I don't see any on my machine

**Comment by rashil2000** [comment]
I see. Excellent manifest btw!

**Pull request merged**